src/viewer/reference/native_plugins.md

```diff
@@ -218,3 +268,21 @@ Parameters are saved in local storage
 
 A plugin as un button that allows to configure door openings and space names.
 Parameters are saved in local storage
+
+# Viewer plugins
+
+It is possible to configure the UI of plugins displayed as viewer (viewer 3d, viewer 2d, viewer plan, viewer dwg, viewer dxf):
+
+| Property      | Description                                                                                                                                |
+| :------------ | :----------------------------------------------------------------------------------------------------------------------------------------- |
+| `modelLoader` | Can be `hidden` or `disabled`. If `hidden`, models will be loaded but the list won't be shown. If `diabled`, the model wont be loaded too. |
```

Comment:
```suggestion
| `modelLoader` | Can be `"hidden"` or `"disabled"`. If `hidden`, the component isn't shown but it will load models defined in the viewer parameters. If `disabled`, the models won't be loaded and you must load them manually |
```